docs/shared/nx-core.md

```diff
@@ -94,27 +107,31 @@ cache.
 In this example, we used a generator to create the package, but you could have also created it by hand or copied it
 from another project.
 
+// Placeholder: talk about project.json/workspace.json default now is using project.json
+
 The change in `workspace.json` is the only thing required to make Nx aware of the `simple` package. As long as you
 include the project in `workspace.json`, Nx will include that project source in its graph computation and source
 code analysis. It will, for instance, analyze the project's source code, and it will know when it can reuse the
 computation from the cache and when it has to recompute it from scratch.
```

Comment:
by default the project.json will be used in new workspaces. should we take time to explain how that works or link out to the configs sections?